setup.cfg

```diff
@@ -1,3 +1,31 @@
+[metadata]
+classifiers =
+    Environment :: Console
+    License :: OSI Approved :: Apache License
+    Intended Audience :: Developers
```

Comment:
also end users I guess, as end users are expected to run `pip3 install pynvim` themselves.

Reply:
Okay I can add `Intended Audience :: End Users/Desktop`